.github/RELEASE_CHECKLIST.md

```diff
@@ -1,7 +1,7 @@
 # Release Checklist
 
 Before releasing ensure that you have:
-- Decided on a release tag. The repo uses a 3 number semantic versioning system and therefore the tag should be of the form v.X.X.X e.g. v2.0.0. This value will be known in the rest of this document as `<RELEASE_TAG>`.
+- Decided on a release tag. The repo uses a 3 number semantic versioning system and therefore the tag should be of the form v.X.X.X e.g. v1.0.0. This value will be known in the rest of this document as `<RELEASE_TAG>`.
```

Comment:
Can you change this sentence to say "decide" at the start, please

Reply:
But it start with you have....